src/Sylius/Bundle/TaxonomyBundle/Resources/translations/validators.be.yml

```
@@ -6,6 +6,8 @@ sylius:
         name:
             max_length: Назва катэгорыі не можа быць даўжэй за 255 сімвалаў.
             not_blank: Калі ласка, увядзіце назву катэгорыі.
+    parent:
```

Bad key indentation:
```suggestion
        parent:
```